packager/media/file/udp_file_posix.cc

```diff
@@ -187,6 +187,9 @@ bool UdpFile::Open() {
   local_sock_addr.sin_family = AF_INET;
   local_sock_addr.sin_port = htons(dest_port);
   local_sock_addr.sin_addr.s_addr  = htonl(dest_addr);
+  // We do not need to require exclusive bind of udp sockets
+  int optval = 1;
```

Comment:
Replace "int" with "const int" to make it clear that it is a const input parameter.